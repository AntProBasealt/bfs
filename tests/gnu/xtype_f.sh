bfs_diff links -xtype f
