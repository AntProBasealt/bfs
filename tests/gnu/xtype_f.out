links/deeply/nested/file
links/deeply/nested/link
links/file
links/hardlink
links/symlink
